### ISSUE TITLE
Add markdown-it (better markdown engine)

### ISSUE THREAD

**Issue by svicalifornia** [jescalan/accord#331]
markdown-it has an impressive set of extensions for customizing the syntax and features of Markdown. Many folks consider it to be a much better engine than marked.

1. Is it possible to add markdown-it as a language option in accord?
2. Is it possible to configure Roots to replace marked with markdown-it for .markdown and .md files?

**Comment by jescalan** [comment]
1. Yes it is
2. I don't see why not

You'd just have to make a pull request to accord to add an engine for markdown-it!

**Comment by svicalifornia** [comment]
Great. But will there be a conflict on the `.markdown` and `.md` extensions with the existing `marked` engine? How will accord resolve those file extensions when a second engine is added on the same extensions?

**Comment by jescalan** [comment]
Yeah, you'd just have to not have marked and markdown-it both in your package.json. With both there, you'd have a conflict, but with one you're set.

**Comment by svicalifornia** [comment]
Ah, that makes sense. :) Thanks!
